packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/cfn-response.ts

```diff
@@ -0,0 +1,112 @@
+// The contents of this file were taken from the AWS CDK provider framework.
```

Comment:
I assume there is no modification for the forked code (If there is any, please note, also same for other forked code from CDK custom resource provider)

Reply:
There is in some files. I'll note what's been removed.

Reply:
Noted modifications in review comments for the individual files.